packages/experimental/Avatar/src/suffixes.ts

```diff
@@ -0,0 +1,107 @@
+export const suffixes = new Set<string>([
+  'Mr',
+  'Mrs',
+  'Miss',
+  'Ms',
+  'Dr',
+  'Admiral',
+  'Air Comm',
+  'Ambassador',
+  'Baron',
+  'Baroness',
+  'Brig & Mrs',
+  'Brig Gen',
+  'Brigadier',
+  'Brother',
+  'Canon',
+  'Capt',
+  'Chief',
+  'Cllr',
+  'Col',
+  'Commander',
+  'Commander & Mrs',
+  'Consul',
+  'Consul General',
+  'Count',
+  'Countess',
+  'Countess of',
+  'Cpl',
+  'Dame',
+  'Deputy',
+  'Dr & Mrs',
+  'Drs',
+  'Duchess',
+  'Duke',
+  'Earl',
+  'Father',
+  'General',
+  'Gräfin',
+  'HE',
+  'HMA',
+  'Her Grace',
+  'His Excellency',
+  'Ing',
```

Comment:
Ing can be a last name for people so I'm not sure if including this would be safe, unless we specifically check that it's appearing before someone's name instead of as a last name

Reply:
I'll remove it